acos_client/v30/file/ssl_cert.py

```diff
@@ -30,8 +30,8 @@ def exists(self, file):
         except acos_errors.NotFound:
             return False
 
-    def _set(self, file=None, cert=None, size=None, certificate_type=None,
-             action=None, update=False, **kwargs):
+    def _set(self, file="", cert="", size="", certificate_type="",
```

Comment:
I know this sounds silly, but we may want to set the blank string as a variable or put in a comment about why this is the way it is.